discord/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 """
 
-__title__ = 'discord'
+__title__ = 'pycord'
```

Comment:
I think this should stay as discord, since it's a module name, right?

Reply:
That’s not how it works that’s the title of the embed thingy it shows iirc

Reply:
We typically replace "discord.py" with py-cord, but leave discord alone since we didn't change the module name. Since this is in __init__.py, I don't think it relates to the docs at all.

Reply:
Okay I should've changed it back with the newest commit